main.py

```diff
@@ -0,0 +1,181 @@
+import json
+
+from flask import Flask, jsonify, request, Response
+from flask_marshmallow import Marshmallow
+from flask_rest_paginate import Pagination
+from flask_sqlalchemy import SQLAlchemy
+
+# from flask_paginate import Pagination, get_page_args
+
+app = Flask(__name__)
+
+####### CONFIGURING DATABASES #######
+
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+app.config['SQLALCHEMY_DATABASE_URI'] = 'mysql://test1:Testing123!@#@localhost/trying'
+
+db = SQLAlchemy(app)
+ma = Marshmallow(app)
+pagination = Pagination(app, db)
+
+
+####### MODELS ##########
+
+class User(db.Model):
+    __tablename__ = 'users'
+
+    id = db.Column(db.Integer, primary_key=True)
+    username = db.Column(db.String(20), unique=True, nullable=False)
+    email = db.Column(db.String(120), unique=True, nullable=False)
+    image_file = db.Column(db.String(20), nullable=False, default='default.jpg')
+    password = db.Column(db.String(60), nullable=False)
+    items = db.relationship('Item', backref='user', lazy='dynamic')
+
+    def __init__(self, username, email, image_file, password):
+        self.username = username
+        self.email = email
+        self.image_file = image_file
+        self.password = password
+
+
+class Item(db.Model):
+    __tablename__ = 'items'
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(120), nullable=False)
+    description = db.Column(db.String(300), nullable=False)
+    user_id = db.Column(db.Integer, db.ForeignKey('users.id'), nullable=False)
+
+    def __init__(self, name, description, user_id):
+        self.name = name
+        self.description = description
+        self.user_id = user_id
+
+    def to_json(self):
+        return {
+            "name": self.name,
+            "description": self.description,
+            "user_id": self.user_id
+        }
+
+
+##### MODELS SCHEMA ######
+class userSchema(ma.Schema):
+    class Meta:
+        fields = ("username", "password")
+
+
+class itemsSchema(ma.Schema):
+    class Meta:
+        fields = ("id", "name", "description", "user_id")
+
+
+users_schema = userSchema()
+items_schema = itemsSchema()
+
+users_schema12 = userSchema(many=True)
+items_schema12 = itemsSchema(many=True)
+
+
+########## ROUTES ##########
+
+@app.route('/Registration', methods=['POST'])
+def register():
+    user_name = request.json['username']
+    email_address = request.json['email']
```

Comment:
If you send a username greater than length 20, an email greater than length 120, or a password greater than length 60, the code will break. Please put the appropriate validations here. Your server code should never break in case the client sends a payload which is not valid.
Remember, client and server are two different entities and server should not rely on client to send the correct payload.

Reply:
please fix similar issues in other APIs as well